pkg/console/subresource/oauthclient/oauthclient.go

```diff
@@ -18,7 +18,7 @@ import (
 // - should look like resourceapply.ApplyService and the other Apply funcs
 //   once its in a trustworthy state, PR to library-go so it can live with
 //   the other Apply funcs
-func ApplyOAuth(client oauthclient.OAuthClientsGetter, required *oauthv1.OAuthClient) (*oauthv1.OAuthClient, bool, error) {
+func CustomApplyOAuth(client oauthclient.OAuthClientsGetter, required *oauthv1.OAuthClient) (*oauthv1.OAuthClient, bool, error) {
```

Comment:
Just renaming this to make it clear, this is not a `library-go` apply func, its a custom impl emulating a `library-go` apply func.